pgdrome/model.py

```diff
@@ -757,15 +757,15 @@ def evaluate(self, fixed_dim, free_dim, coord, attri):
                 coord,
             )
 
-        # only possible if freeDims are one dimensional
-        for i in range(len(free_dim)):
-            if (
-                sum(self.mesh[free_dim[i]].dataY) != 0
-                and sum(self.mesh[free_dim[i]].dataZ) != 0
-            ):
-                raise ValueError(
-                    "free Dimensions are not 1D, interpolation not possible"
-                )
+        # # only possible if free_dims are one dimensional
+        # for i in range(len(free_dim)):
```

Comment:
If it's makes no sense anymore, you can also fully delete that. Maybe reduces confusions in the future :)